### PR TITLE
Added more quality content

### PR DESCRIPTION
**Pull request by tomzorz**

### CODE REVIEW

File: questions.json [hunk 1]
```diff
@@ -13,6 +14,22 @@
     "What is your least favorite friend's middle name?",
     "What is a question an oyster would ask a fish?",
     "What would your full name be if you married your first crush?",
-    "What caused you to realize that capitalism is a sham?"
+    "What caused you to realize that capitalism is a sham?",
+    "What is your favorite security question?",
+    "What are you wearing at time time of setting up the security question?",
```
Comment (cassidoo) [inline questions.json:19]: Typo here, "time time," this one might be kind of difficult to remember the answer to, also

File: questions.json [hunk 2]
```diff
@@ -13,6 +14,22 @@
...
+    "What TV show do you keep rewatching to suppress the anxious feelings of the uncertainty of real life?",
+    "What was the name of your favorite unpaid internship?",
+    "What is the name of your favorite canceled TV show?",
+    "On which street did you lose your childlike sense of wonder?",
+    "In what year did you abandon your dreams?",
+    "When did you stop trying?",
+    "What is the phone number of your first crush that you still remember from 2007?",
+    "Which household appliance do you hit the most often?",
+    "Why is it always raining?",
+    "You should have said something. Back then! But what?"
```
Comment (cassidoo) [inline questions.json:33]: Maybe reword this to, "You should have said something back then. What should you have said?"

File: questions.json [hunk 3]
```diff
@@ -13,6 +14,22 @@
...
+    "What are you wearing at time time of setting up the security question?",
+    "What is your password?",
+    "Which college class did finally break you?"
+    "What is the maiden name of your father's mistress?",
+    "At what age did your childhood pet run away?",
+    "What TV show do you keep rewatching to suppress the anxious feelings of the uncertainty of real life?",
+    "What was the name of your favorite unpaid internship?",
+    "What is the name of your favorite canceled TV show?",
+    "On which street did you lose your childlike sense of wonder?",
+    "In what year did you abandon your dreams?",
```
Comment (cassidoo) [inline questions.json:28]: This might be easy to guess because chances are it's 2008

**Comment by andrewfleer** [comment]
LGTM

File: questions.json [hunk 4]
```diff
@@ -13,6 +14,22 @@
...
+    "When did you stop trying?",
+    "What is the phone number of your first crush that you still remember from 2007?",
```
Comment (cassidoo) [inline questions.json:30]: First crush and 2007 might not line up, maybe just "your crush" and not "first"

**Review by cassidoo — CHANGES_REQUESTED** [review]
Love it, minor changes and notes

File: questions.json [hunk 5]
```diff
@@ -13,6 +14,21 @@
     "What is your least favorite friend's middle name?",
     "What is a question an oyster would ask a fish?",
     "What would your full name be if you married your first crush?",
-    "What caused you to realize that capitalism is a sham?"
+    "What caused you to realize that capitalism is a sham?",
+    "What is your favorite security question?",
+    "What are you wearing at the time of setting up the security question?",
```
Comment (cassidoo) [inline questions.json:19]: I think cut this one

File: questions.json [hunk 6]
```diff
@@ -13,6 +14,21 @@
...
+    "What is your password?",
+    "Which college class did finally break you?"
```
Comment (cassidoo) [inline questions.json:21]: "Which college class broke you?"

**Review by cassidoo — APPROVED** [review]

**Comment by tomzorz** [comment]
weee
